src/nvme/tree.h

```diff
@@ -452,6 +452,14 @@ const char *nvme_ns_get_sysfs_dir(nvme_ns_t n);
  */
 const char *nvme_ns_get_name(nvme_ns_t n);
 
+/**
+ * nvme_ns_get_chrdev_name() -
+ * @n:
+ *
+ * Return: 
```

Comment:
I know so far we are lacking a lot of documentation. But let's try to add some documentation to newly introduced APIs.

Reply:
Got it! let me fill this up! Thanks.